README.md

```diff
@@ -67,4 +67,9 @@ $ cp .env.oss .env
 
         $ yarn sync-schema
 
-* There are some suggested VSCode extensions in `.vscode/extensions.json` and additional docs at [docs/vscode.md](docs/vscode.md).
+* [Circle CI](https://circleci.com/docs/1.0/npm-continuous-deployment/) is set up to publish reaction-force in consistent and reliable way. To update the package version and deploy it on npm. You can read more about publishing and updating npm packages [here](https://docs.npmjs.com/getting-started/publishing-npm-packages):
```

Comment:
Sort of weird grammar I think, we can flush it out a bit more and make it friendlier I think, how about:

"Circle CI is set up to publish reaction-force in a consistent and reliable way. The following commands will update the package version and deploy it to NPM. The package.json file will be updated and a commit staged. When a tag is pushed to `master`, Circle CI will publish to NPM. See [circle.yml]() and the [npm docs]()."